core/validation/specific_validation/pathfinders/round_1.py

```diff
@@ -371,3 +376,92 @@ def _check_actual_forecast_reporting_period(extracted_table_dfs: dict[str, pd.Da
                     )
                 )
     return error_messages
+
+
+def _check_dropdown_values(
+    df: pd.DataFrame, value_column: str, allowed_values: list[str], sheet: str, section: str, error_description: str
+) -> list[Message]:
+    """
+    Check that values in a DataFrame column belong to a list of allowed values.
+    """
+    breaching_row_indices = df[~df[value_column].isin(allowed_values)].index.tolist()
+    breaching_values = df.loc[breaching_row_indices, value_column].tolist()
+    return [
+        _error_message(
+            sheet=sheet,
+            section=section,
+            description=error_description.format(value=value),
+        )
```

Comment:
I think we can return the  ```cell_index``` alongside this information given we already have the ```value_column``` and can therefore know the letter of the column in the Excel sheet, and all we need to do is increment the indices by one to get the row. Hopefully it's not more effort than that

Reply:
Fair point, I was going to make a separate ticket for this but makes more sense to add it here if it's a simple change